xml/ay_bootloader.xml

```diff
@@ -96,30 +96,29 @@
   <tip>
     <title>Hibernation</title>
     <para>
-     If there is a need for specific hibernation settings then
-     <literal>resume</literal> or <literal>noresume</literal> in
-     <literal>append</literal> configuration can be used. It has specific
-     handling.
+     If there is a need for specific hibernation settings, then
+     <literal>resume</literal> or <literal>noresume</literal> in the
+     <literal>append</literal> configuration can be used.
     </para>
     <para>
-      To disable hibernation regardless what installation proposes then specify
-      <literal>noresume</literal> as kernel parameter in
+      To disable hibernation regardless of what the installer proposes,
+      specify <literal>noresume</literal> as a kernel parameter in the
       <literal>append</literal> section.
     </para>
     <para>
-      To specify hibernation device use <literal>resume</literal> key with
-      value which is device path. Recommended way to get stable result is using
-      own partitioning and having swap device with label. Example with labeled
-      swap:
+      To specify the hibernation device, use the <literal>resume</literal>
+      key with the device path. The recommended way to get stable results is
+      configuring your own partitioning, and having a swap device with a
+      label:
       <screen>
         &lt;append&gt;quiet resume=/dev/disk/by-label/my_swap&lt;/append&gt;
       </screen>
-    </para>
+     </para>
     <para>
-      If <literal>resume</literal> neither <literal>noresume</literal> is used
-      or if resume contains device that won't exist on installed system then
-      hibernation is proposed by installer, either enabled or disabled and also
-      hibernation device will be proposed.
+      If you do not use <literal>resume</literal> or
+      <literal>noresume</literal>, or if <literal>resume</literal> specifies
+      a device that will not exist on the installed system, then
+      hibernation will be proposed by installer, either enabled or disabled, and the installer will also propose a hibernation device.
```

Comment:
when I read it I probable use not so clear wording here. As with disabled hibernation it can mean that installer propose noresume, but in fact in such case installer does not add any resume or noresume. Not sure how to write it to make it obvious.

Reply:
Thanks, this needs to be clear. Is this correct:

"If you do not use resume or
      noresume, or if resume specifies
      a device that will not exist on the installed system, then
      noresume will be proposed by installer. The installer will not make any changes unless you approve them."

Reply:
I think it is better. Just last sentence is not correct. It will do change and confirm depends on confirm mode. So to keep it simple, I would remove the last sentence as it is complicated enough :)

Reply:
maybe I would slightly change wording to:

"If you do not use resume or
noresume, or if resume specifies
a device that will not exist on the installed system, then
the installer may propose a correct value for resume=, or it may remove hibernation parameter completely, depending on installer logic."

What do you think?

Reply:
I think I am happy that the AY team maintains the manual 😁
That sounds good, short and to the point.